### PR TITLE
Feature/edit profile loader

### PR DESCRIPTION
**Pull request by sanjayradadiya**
Fixes #

Changes in this pull request:
- fix edit profile loader issue
- fix change password showing error `please enter same password` message after successfully save password
- add notification manager for success or error message 

@norbertschuler @harshvitra

### CODE REVIEW

File: app/components/EditUserProfile/index.native.js [hunk 1]
```diff
@@ -132,6 +132,7 @@ export default class EditUserProfile extends Component {
     let schemaOptions = parsedSchema[userType][profileType].schemaOptions;
     if (profileType == 'password') {
       try {
+        console.log('schemaOptionm', schemaOptions);
```
Comment (norbertschuler) [inline app/components/EditUserProfile/index.native.js:135]: please remove before merging

File: app/components/Common/Button/PrimaryButton.native.js [hunk 2]
```diff
@@ -2,22 +2,34 @@ import React from 'react';
...
     style={[styles.primaryButton, buttonStyle]}
+    disabled={loading}
   >
     <View style={styles.textContainer}>
-      <Image source={image} style={image ? styles.image : { width: 0 }} />
-      <Text style={[styles.primaryButtonText, textStyle]}>{children}</Text>
+      {loading ? (
+        <ActivityIndicator
+          size="large"
+          color='white'
```
Comment (norbertschuler) [inline app/components/Common/Button/PrimaryButton.native.js:24]: check `npm run lint:errors`

File: app/components/Common/Button/PrimaryButton.native.js [hunk 3]
```diff
@@ -2,22 +2,34 @@ import React from 'react';
...
     activeOpacity={0.6}
     onPress={() => onClick()}
     style={[styles.primaryButton, buttonStyle]}
+    disabled={loading}
   >
     <View style={styles.textContainer}>
-      <Image source={image} style={image ? styles.image : { width: 0 }} />
-      <Text style={[styles.primaryButtonText, textStyle]}>{children}</Text>
+      {loading ? (
+        <ActivityIndicator
```
Comment (norbertschuler) [inline app/components/Common/Button/PrimaryButton.native.js:22]: This is a nice solution, but it should be applied to every usage of `PrimaryButton` in the whole app. And it's a pity some button are not any more using the `PrimaryButton` component, e.g. in the native login form where the ActivityIndicator instead is _selfmade_. This does not give our software architecture very much consistency.

Comment (sanjayradadiya) [reply app/components/Common/Button/PrimaryButton.native.js:22]: @norbertschuler should I need to remove this ?

Comment (harshvitra) [reply app/components/Common/Button/PrimaryButton.native.js:22]: @norbertschuler we need to add the consistency, we will soon work on this.

**Comment by harshvitra** [comment]
> Some comments. And no, you alone can not refactor the whole app. We must communicate this with the whole team.

I don't think it will affect any other part of the app unless the 'loading' props is passed from there. 
So this solution can work for Edit Profile

**Comment by norbertschuler** [comment]
> > Some comments. And no, you alone can not refactor the whole app. We must communicate this with the whole team.
> 
> I don't think it will affect any other part of the app unless the 'loading' props is passed from there.
> So this solution can work for Edit Profile

O.k, as I have noted the refactoring request on https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1837, we can merge this.

**Pull request merged**